nautobot/core/management/commands/build_ui.py

```diff
@@ -128,6 +128,11 @@ def handle(self, *args, **options):
         }
         loglevel = f"--loglevel {verbosity_map[verbosity]}"
 
+        self.run_command(
+            f"cp -r {Path(settings.BASE_DIR, 'ui')}/. {Path(settings.NAUTOBOT_UI_DIR)}",
```

Comment:
TODO: Make a directory if it doesn't exist, which is likely in `pip install` case.